### DISCUSSION TITLE
未来是否可以增加视频的语音翻译功能?

### DISCUSSION THREAD

**Discussion by ccdanpian** [immersive-translate/immersive-translate#1485] [Ideas]
### 新功能 | New Feature

1.实时视频语音翻译，可以由用自己配置开或关

### 为什么您想要添加这些功能 | Why you want to add these features

1. 可以用在在不方便或不想一直看视频字幕的时候。

**Comment by theowenyoung** [comment]
指的是本地视频吗？还是 youtube 上的视频？

**Reply by ccdanpian** [reply]
youtube上的视频，当然如果本地视频能一起支持就更好了。

**Reply by theowenyoung** [reply]
现在就支持呀，试试这样开启： https://immersivetranslate.com/docs/usage/#youtube-%E5%8F%8C%E8%AF%AD%E5%AD%97%E5%B9%95

**Reply by ccdanpian** [reply]
现在已经支持语音了吗？我昨天在手机Firefox上安装的插件，已经开了这个开关，语音还是英语，只是字幕有双语。
